istio/datadog_checks/istio/metrics.py

```diff
@@ -378,15 +398,59 @@
     'sidecar_injection_success_total': 'sidecar_injection.success_total',
     'sidecar_injection_failure_total': 'sidecar_injection.failure_total',
     'sidecar_injection_skip_total': 'sidecar_injection.skip_total',
+    ## Unknown
```

Comment:
Nit: what about this metric is unknown?

Reply:
I wasn't able to find the metric type for these. I just wanted to document them so that in the future we can come back and validate these should we happen to get a curl output that does have these metrics.